src/utils.js

```diff
@@ -7,26 +9,19 @@ export const noop = () => {};
 
 export const apiRequest = (url, accessToken, options = {}) => {
   const allOptions = {
+    ...options,
+    url,
     headers: {
-      Authorization: `Bearer ${accessToken}`,
-      'Content-Type': 'application/vnd.api+json',
-      Accept: 'application/vnd.api+json'
-    },
-    ...options
+      ...options.headers,
+      'Content-Type': 'application/vnd.api+json'
+    }
   };
```

Comment:
I think it would be more readable to use Immutable:

```js
const allOptions = Imm.fromJS(options)
  .set('url', url)
  .setIn(['headers', 'Content-Type'], 'application/vnd.api+json')
  .toJS();
```